### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by mrdudz** [Panda381/PicoVGA#19]
What License applies to this code? Is it ok to use in a commercial/closed source context?

**Comment by Panda381** [comment]
Yes, the code can be used freely, for commercial or non-commercial purposes. But I want to warn you that I no longer support this library because it is too difficult to use. And it is advisable to make a modification to it to add HSYNC+VSYNC, because some monitors do not support composite CSYNC. I prefer to use the PicoLibSDK library, where the display driver is simpler, using a simple frame buffer. https://github.com/Panda381/PicoLibSDK

**Comment by mrdudz** [comment]
Thanks for quick answer!

It would be great if you could a license text, or link to it in the readme, though :)
